### ISSUE TITLE
cannot transfer ephys data to local server

### ISSUE THREAD

**Issue by karolinasocha** [int-brain-lab/iblrig#294]
During transferring ephys data from acquisition computer to the local server I have this kind of error message. I followed all steps before (including creating empty file: transfer_me.flag)
```

(iblenv) C:\iblscripts\deploy\ephyspc>
(iblenv) C:\iblscripts\deploy\ephyspc>python transfer_ephys_session.py D:\SGL_DATA\Subjects\ Y:\Subjects\
usage: transfer_ephys_session.py [-h] [-l LOCAL] [-r REMOTE] [-f]
transfer_ephys_session.py: error: unrecognized arguments: D:\SGL_DATA\Subjects\ Y:\Subjects\
```
it is the same when I include output folders and destination folder:
```

(iblenv) C:\iblscripts\deploy\ephyspc>python transfer_ephys_session.py D:\SGL_DATA\Subjects\KS003\2019-11-20 Y:\Subjects\KS003\2019-11-20
usage: transfer_ephys_session.py [-h] [-l LOCAL] [-r REMOTE] [-f]
transfer_ephys_session.py: error: unrecognized arguments: D:\SGL_DATA\Subjects\KS003\2019-11-20 Y:\Subjects\KS003\2019-11-20
```

**Comment by GaelleChapuis** [comment]
@oliche does this need double \\ in the path ?

**Comment by karolinasocha** [comment]
when I write only `python transfer_ephys_session.py`, I have this message:

```
(iblenv) C:\iblscripts\deploy\ephyspc>python transfer_ephys_session.py
Traceback (most recent call last):
  File "transfer_ephys_session.py", line 26, in <module>
    iblscripts_folder=iblscripts_folder)
  File "C:\Users\IBL_ephys\Miniconda3\envs\iblenv\lib\site-packages\ibllib\pipes\misc.py", line 212, in confirm_ephys_remote_folder
    remote_folder = pars['REMOTE_DATA_FOLDER_PATH']
KeyError: 'REMOTE_DATA_FOLDER_PATH'
```

**Comment by karolinasocha** [comment]
And this is error that I have when I specify local folder and remote : 
```

(iblenv) C:\iblscripts\deploy\ephyspc>python transfer_ephys_session.py -l D:/SGL_DATA/Subjects/KS003/2019-11-20 -r Y:/Subjects/KS003/2019-11-20
2019-11-21 10:14:23.608 ERROR    [folders.py:29] No "Subjects" folder in children folders
Traceback (most recent call last):
  File "transfer_ephys_session.py", line 26, in <module>
    iblscripts_folder=iblscripts_folder)
  File "C:\Users\IBL_ephys\Miniconda3\envs\iblenv\lib\site-packages\ibllib\pipes\misc.py", line 216, in confirm_ephys_remote_folder
    local_folder = folders.subjects_data_folder(local_folder, rglob=True)
  File "C:\Users\IBL_ephys\Miniconda3\envs\iblenv\lib\site-packages\alf\folders.py", line 30, in subjects_data_folder
    raise(ValueError)
ValueError
```

**Comment by nbonacchi** [comment]
You now need to specify the local and remote folder in the params folder.
Try `python ephyspc_params_file.py`

**Comment by karolinasocha** [comment]
I defined location of files: python transfer_ephys_session.py -l D:\\SGL_DATA\\Subjects -r Y:\\Subjects

but did not find folder: FileNotFoundError: [Errno 2] No such file or directory: 
```

(iblenv) C:\iblscripts\deploy\ephyspc>python transfer_ephys_session.py -l D:\\SGL_DATA\\Subjects -r Y:\\Subjects
LOCAL: D:\SGL_DATA\Subjects
REMOTE: Y:\Subjects

Found session: D:\SGL_DATA\Subjects\KS003\2019-11-21\001\raw_ephys_data
Traceback (most recent call last):
  File "C:\Users\IBL_ephys\Miniconda3\envs\iblenv\lib\shutil.py", line 550, in move
    os.rename(src, real_dst)
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'D:\\SGL_DATA\\Subjects\\KS003\\2019-11-21\\001\\raw_ephys_data\\probe00\\_spikeglx_ephysData_g0_t0.imec0.ap.bin' -> 'D:\\SGL_DATA\\Subjects\\KS003\\2019-11-21\\001\\raw_ephys_data\\raw_ephys_data\\probe00\\_spikeglx_ephysData_g0_t0.imec0.ap.bin'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "transfer_ephys_session.py", line 26, in <module>
    iblscripts_folder=iblscripts_folder)
  File "C:\Users\IBL_ephys\Miniconda3\envs\iblenv\lib\site-packages\ibllib\pipes\misc.py", line 232, in confirm_ephys_remote_folder
    move_ephys_files(str(session_path))
  File "C:\Users\IBL_ephys\Miniconda3\envs\iblenv\lib\site-packages\ibllib\pipes\misc.py", line 376, in move_ephys_files
    shutil.move(str(i0f), str(probe00_path / i0f.name))
  File "C:\Users\IBL_ephys\Miniconda3\envs\iblenv\lib\shutil.py", line 564, in move
    copy_function(src, real_dst)
  File "C:\Users\IBL_ephys\Miniconda3\envs\iblenv\lib\shutil.py", line 263, in copy2
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "C:\Users\IBL_ephys\Miniconda3\envs\iblenv\lib\shutil.py", line 121, in copyfile
    with open(dst, 'wb') as fdst:
FileNotFoundError: [Errno 2] No such file or directory: 'D:\\SGL_DATA\\Subjects\\KS003\\2019-11-21\\001\\raw_ephys_data\\raw_ephys_data\\probe00\\_spikeglx_ephysData_g0_t0.imec0.ap.bin'

(iblenv) C:\iblscripts\deploy\ephyspc>python transfer_ephys_session.py -l D:\\SGL_DATA\\Subjects -r Y:\\Subjects
LOCAL: D:\SGL_DATA\Subjects
REMOTE: Y:\Subjects

Found session: D:\SGL_DATA\Subjects\KS003\2019-11-20\001
Traceback (most recent call last):
  File "transfer_ephys_session.py", line 26, in <module>
    iblscripts_folder=iblscripts_folder)
  File "C:\Users\IBL_ephys\Miniconda3\envs\iblenv\lib\site-packages\ibllib\pipes\misc.py", line 234, in confirm_ephys_remote_folder
    copy_wiring_files(str(session_path), iblscripts_folder)
  File "C:\Users\IBL_ephys\Miniconda3\envs\iblenv\lib\site-packages\ibllib\pipes\misc.py", line 421, in copy_wiring_files
    if PARAMS['PROBE_TYPE_00'] != PARAMS['PROBE_TYPE_01']:
KeyError: 'PROBE_TYPE_00'
```

**Comment by nbonacchi** [comment]
Delete the params file and recreate, you're missing a couple of params

**Comment by karolinasocha** [comment]
I deleted parms file and run transfer_ephys_session.py again:

```
(iblenv) C:\iblscripts\deploy\ephyspc>python transfer_ephys_session.py
Where's your LOCAL 'Subjects' data folder? [default: D:\iblrig_data\Subjects]:
Where's your REMOTE 'Subjects' data folder? [default: \\iblserver.champalimaud.pt\ibldata\Subjects]: Y:\Subjects
What's the type of PROBE 00? ([3A] | 3B)> 3B
What's the type of PROBE 01? ([3A] | 3B)> 3B
Created C:\Users\IBL_ephys\AppData\Roaming\.ephyspc_params
{'DATA_FOLDER_PATH': 'D:\\iblrig_data\\Subjects', 'REMOTE_DATA_FOLDER_PATH': 'Y:\\Subjects', 'PROBE_TYPE_00': '3B', 'PROBE_TYPE_01': '3B'}
LOCAL: D:\iblrig_data\Subjects
REMOTE: Y:\Subjects
Nothing to transfer, exiting...

(iblenv) C:\iblscripts\deploy\ephyspc>python transfer_ephys_session.py
LOCAL: D:\iblrig_data\Subjects
REMOTE: Y:\Subjects
Nothing to transfer, exiting...

(iblenv) C:\iblscripts\deploy\ephyspc>``

**Comment by karolinasocha** [comment]
success! to summarise: the issue was related to the wrong information in params file that is located in Roaming/ephyspc_params. If you run into similar issues as I did, delete ephyspc_params from that folder and run python transfer_ephys_session.py it will ask you for local and remote paths and then will save in that folder (example below for 3B system).  I run into troubles with the transfer because that file existed before but did not have full information. I also did not realise that this step should be done before trying to transfer actual data.
Example of ephyspc_params: 
![image](https://user-images.githubusercontent.com/32096962/69434754-2944ef00-0d36-11ea-852f-b8cbd984a94f.png)

Example of successful transfer: 
![image](https://user-images.githubusercontent.com/32096962/69434801-41b50980-0d36-11ea-884e-08d1684c2071.png)
